src/app/shared/animations.ts

```diff
@@ -9,6 +9,13 @@ const fadeInOut = [
     transition("in => out", [animate("0.5s")]),
     // todo - could use :enter and :exit properties also (need to confirm doesn't break existing functionality)
   ]),
+  trigger("noFade", [
+    state("in", style({ opacity: 1 })),
+    state("out", style({ opacity: 0 })),
```

Comment:
Do we still need these `state()` functions if we're not actually animating? You could try removing them and see if it still works. If not, we may be able to remove the `style()` functions, or at least the reference to opacity, as we're not interested in that property.

Reply:
Realised that I didn't need state(out) but did need state(in). The style is just for the disappearing, I believe.